src/transcoding/codec/codecs/libs/vaapi.c

```diff
@@ -310,14 +636,14 @@ static const codec_profile_class_t codec_profile_vaapi_hevc_class = {
         .ic_properties = (const property_t[]){
             {
                 .type     = PT_INT,
-                .id       = "tier",
+                .id       = "tier",     // Don't change
                 .name     = N_("Tier"),
-                .desc     = N_("Set tier (general_tier_flag) [0=main 1=high]"),
+                .desc     = N_("Set tier (-1, 0 or 1) [-1=skip 0=main 1=high]"),
```

Comment:
This should probably be a combo instead, allowing to choose between "" (empty), "main" and "high".

Reply:
I agree with you; can we take that action for next release? I plan to add more h264 profiles (4:2:2, 4:4:4) . In terms of priority I think the next few things are higher priority.